index.html

```diff
@@ -14,6 +14,21 @@
       <div id="ui">
 	<div id="sidebar">
 	  <div id="controls">
+      <h3>Date Range</h3>
+      <form name="dateSelection">
+        <div class="dateRadio">
+          <input type="radio" name="radioDate" id="radioMonth" checked>
```

Comment:
I did try to do this, but I think because of how the html document/script is laid out the functions are out of scope, as I'm getting `Uncaught ReferenceError: displayPastMonth is not defined`.